tsinfer/formats.py

```
@@ -243,21 +243,32 @@ def zarr_summary(array):
     return ret
 
 
-def chunk_iterator(array, indexes=None):
+def chunk_iterator(array, indexes=None, mask=None):
     """
     Utility to iterate over closely spaced rows in the specified array efficiently
     by accessing one chunk at a time (normally used as an iterator over each row)
     """
+    if mask is not None:
+        assert len(mask) == len(array)
+
```

Maybe simpler to create default mask (keep everything)?